samples/remote_attestation/common/attestation.h

```diff
@@ -18,7 +18,7 @@ class Attestation
     Attestation(Crypto* crypto, uint8_t* enclave_mrsigner);
 
     // Generate remote evidence for the given data.
-    bool generate_remote_evidence(
+    bool generate_remote_attestation_evidence(
         const uint8_t* data,
         size_t data_size,
         uint8_t** remote_evidence_buf,
```

Comment:
```suggestion
        uint8_t** remote_evidence_buffer,
```
(and same in line 25.  I know you didn't change this, but I see you made the same fix elsewhere, thanks for that)